gradescopeCronJob/gradescope_to_spreadsheet.py

```diff
@@ -228,6 +229,71 @@ def make_batch_request(sheet_api_instance):
     make_request(batch_request)
     logger.info(f"Completing batch request")
 
```

Comment:
In the function `retrieve_grades_from_gradescope`, can you add a logging statement that will log if retrieving the grades from gradescope succeeded or failed?